bitnami/grafana-operator/values.yaml

```diff
@@ -428,7 +423,7 @@ grafana:
   image:
     registry: docker.io
     repository: bitnami/grafana
-    tag: 9.5.2-debian-11-r6
+    tag: 9.5.3-debian-11-r3
     digest: ""
```

Comment:
We are no longer using these parameters right? We should review the useless parameters and remove them both from the templates and `values.yaml`